### ISSUE TITLE
Algolia search Tutorial links broken 

### ISSUE THREAD

**Issue by clairefro** [redwoodjs/redwoodjs.com#648]
#### Problem
Algolia search results for "Tutorial" items are pointing to the old (non-existent) tutorial `.html` pages, causing a "Page not Found" error as tutorial content is no longer hosted on .com since moving to "learn" repo. Netlify redirects are not picking up on the redirect because of the `.html` suffix from Algolia indexing

#### Possible solutions 
1. Remove tutorial content from Algolia indexing for .com
- PRO: clean separation between .com and learn
- CON: loss of discover-ability of helpful content
2. Add duplicate redirects for all tutorial pages, adding `.html` suffix 
- PRO: greater discover-ability of helpful content
- CON: confusing to be redirected off .com to "learn" site; content could become stale as "learn" docs evolve

example of option 2:
current
```toml
# netlify.toml
[[redirects]]
  from = "/tutorial/redwood-file-structure"
  to = "https://learn.redwoodjs.com/docs/tutorial/redwood-file-structure"
  status = 301
  force = true
```
suggested (redirects cover both "naked" and `.html` urls)
```toml
# netlify.toml
[[redirects]]
  from = "/tutorial/redwood-file-structure"
  to = "https://learn.redwoodjs.com/docs/tutorial/redwood-file-structure"
  status = 301
  force = true

[[redirects]]
  from = "/tutorial/redwood-file-structure.html"
  to = "https://learn.redwoodjs.com/docs/tutorial/redwood-file-structure"
  status = 301
  force = true
```
#### Reproduction
1. On .com, search "home" in search bar
2. Select any search result from the "Tutorial" section
3. No redirect made, instead see 404 page on .com

![image](https://user-images.githubusercontent.com/9841162/112763848-0c1fd080-8fbb-11eb-80c8-962e6b8dd137.png)

![image](https://user-images.githubusercontent.com/9841162/112763853-10e48480-8fbb-11eb-8b58-62cad3c9b91e.png)

**Comment by clairefro** [comment]
EDIT: The `.html` redirects should exist either way, in case users have bookmarked tutorial pages found via Algolia (as pointed out by @Tobbe)

**Comment by cannikin** [comment]
I can delete the Algolia index and then deploy, which will eliminate the Tutorial results?

**Comment by clairefro** [comment]
I believe it would. To be safe, we can keep a backup of the current record base by clicking "Duplicate" for the current index in Algolia

**Comment by clairefro** [comment]
resolved by #649

**Comment by cannikin** [comment]
> To be safe, we can keep a backup of the current record base by clicking "Duplicate" for the current index in Algolia

I should have checked this issue before I deleted it, sorry! 😬 We could re-generate if we really needed to, but I think we'll be okay.

![image](https://media.giphy.com/media/2zoFXIZs5F6tSPs8fB/giphy.gif)
